### PR TITLE
Cygwin support

### PR DESCRIPTION
**Pull request by panyamin**
Hello,

Maybe an ugly fix, but it's working! :)

### CODE REVIEW

File: lib/execjs/external_runtime.rb [hunk 1]
```diff
@@ -28,8 +28,15 @@ def exec(source, options = {})
         source = @runtime.compile_source(source)
 
         tmpfile = write_to_tempfile(source)
+
+        if ExecJS.cygwin?
+          filepath = `cygpath -m #{tmpfile.path}`.rstrip
```
Comment (rafaelfranca) [inline lib/execjs/external_runtime.rb]: Could we use the `-w` options instead of `-m`? It is better documented.

Comment (panyamin) [inline lib/execjs/external_runtime.rb]: Well, I'll try if it works.
I'll keep you updated asap.

Comment (panyamin) [inline lib/execjs/external_runtime.rb]: **Update**
Even if it's working with `-w`, path generated is (*note the backslashes*):
`C:\CYGWIN~1\path/to/something`

And that a bit ugly, while `-m` generate that:
`C:/CYGWIN~1/path/to/something`

What do you think?

Comment (rafaelfranca) [inline lib/execjs/external_runtime.rb]: Yeah, seems good to keep `-m`

Comment (panyamin) [inline lib/execjs/external_runtime.rb]: Awesome! Thank you ;)

**Pull request merged**